source/agora/consensus/Quorum.d

```diff
@@ -251,7 +314,7 @@ private const(PublicKey)[] getKeys (size_t count)
 }
 
 /// Create a shorthash from a 64-byte blob for RNG initialization
```

Comment:
Just a note: the comment was previously wrong (it was 32 bytes), but now it's correct.